src/main/java/com/example/crm/ResourceManager/ResourceManagerRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.crm.ResourceManager;
+
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ResourceManagerRepository extends JpaRepository<ResourceManager, Long> {
+
+    Optional<ResourceManager> findResourceManagerByEmail(String email);
```

Comment:
I don't think we need this method, this is implicit when we use @Repository annotation. Try removing this method and see if it still works.